ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>"Feature Store organizes data with the following..." (capitalize Store)</p><p> </p><p>"you will create a featurestore" (use featurestore one word)</p><p> </p><p>"The Users entity type has the age, gender, and like genres features." (add "type" "the" and "and")</p><p> </p><p>"The Movies entity type has the genres and average rating features." (similar change as above)</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>